### DISCUSSION TITLE
How can I rescale a KSPI-E Engine?

### DISCUSSION THREAD

**Discussion by isdi0** [TweakScale/TweakScale#305] [Support] [answered]
I'm playing KSP with a bunch of mods, including KSPI-E (and TweakScale, obviously), and for some reason the engines that I've tried using are unable to be rescaled. I've tried it with stock engines, and they work fine, but for KSPI-E engines, the part window displays "TweakScale is: Unavailable"
I'm not 100% sure this is a bug, which is why I'm posting this here.

**Comment by Lisias** [comment, marked as answer]
Hi.

Unfortunately, there's a glitch on the TweakScale support for Waterfall that makes the plumes to reset to the default sizes when you do a Revert to Launch. This was causing problems to KSPI-E's maintainer, that choose to remove TweakScale from all the engines in consequence

So, there's nothing I can do, as this is a patch applied by KSPI-E and it's out of my reach to do something about.

You can reach KSPI-E maintainer and ask him what to do to prevent removing TweakScale support from engines.

(a fix for the problem is in the backlog for some time - I didn't managed, yet, to understand why the plumes are reset on a Revert to Launch, but not on a Launch or Vessel Switch…)

**Comment by Lisias** [comment]
For the sake of completude, I just fired up 1.12.5 with Waterfall and TweakScale. The thing is working:

![screenshot14](https://github.com/TweakScale/TweakScale/assets/64334/f5aa7f6d-50b6-464b-a9e7-e8c9a685dec1)

But still with the Revert to Launch glitch:

![screenshot15](https://github.com/TweakScale/TweakScale/assets/64334/9eb74eb0-a1ad-404e-b36b-f2a588c4b8c4)

Switching vessels don't work, the only way to fix the plumes is to switching Scenes.
